__test__/getFavourite.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require('supertest');
+const app = require('../server/app');
```

Comment:
please add an empty line here to separate your modules from the third-party modules